src/helpers.ts

```diff
@@ -192,54 +221,54 @@ export const getRouterSchemaImportByOpName = (
   const inputType = getInputTypeByOpName(opType, modelName);
 
   return inputType
-    ? `import { ${inputType} } from "../schemas/${opType}${modelName}.schema";`
+    ? `import { ${inputType} } from "../schemas/${opType}${modelName}.schema"; `
     : '';
 };
 
 export const getInputTypeByOpName = (opName: string, modelName: string) => {
   let inputType;
   switch (opName) {
     case 'findUnique':
-      inputType = `${modelName}FindUniqueSchema`;
+      inputType = `${modelName} FindUniqueSchema`;
```

Comment:
These spaces are messing up Schema names

Reply:
Oh sorry didn't notice that between all the whitespace changes 😓